operator/v1/types_console.go

```diff
@@ -21,13 +21,27 @@ type ConsoleSpec struct {
 	// customization is used to optionally provide a small set of
 	// customization options to the web console.
 	// +optional
-	Customization ConsoleCustomization `json:"customization"`
+	Customization ConsoleCustomization `json:"customization,omitempty"`
```

Comment:
No need to add `omitempty` here, since golang does not have notion of empty struct it will still render it, unless it's pointer to a struct.